cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/EdgeCachePass.kt

```diff
@@ -82,6 +83,13 @@ object Edges {
  *
  * The cache itself is stored in the [Edges] object.
  */
+@DependsOn(TypeResolver::class)
+@DependsOn(TypeHierarchyResolver::class)
+@DependsOn(EvaluationOrderGraphPass::class)
+@DependsOn(SymbolResolver::class)
+@DependsOn(DFGPass::class)
+@DependsOn(DynamicInvokeResolver::class)
+@DependsOn(ControlFlowSensitiveDFGPass::class)
```

Comment:
I would only include EvaluationOrderGraphPass, SymbolResolver, and the DFG passes here. Those "should" be enough